packages/apps/docs/src/scripts/importReadme.mjs

```diff
@@ -119,6 +119,9 @@ const recreateUrl = (pages, url, root) => {
         url = `${url}#${slug}`;
       }
 
+      // remove double slashes from url, if any
+      url = url.replace(/\/\//g, '/');
```

Comment:
what is happening here?

Reply:
When adding the Next.js Link component, some of the internal links break due to "//" in their paths when running `rush build`:
```
Invalid href '/docs/kadena/client//signing#integrated-sign-request-to-chainweaver-desktop' passed to next/router in page: '/docs/kadena/client/getting-started'. Repeated forward-slashes (//) or backslashes \ are not valid in the href.
```

Adding a check to avoid running this replacement if `http` is found in the string